### PR TITLE
Volunteer view notifications

### PR DESCRIPTION
**Pull request by noureldin-khaled**
User can view his/her notifications and mark any of them as unread.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -3,9 +3,9 @@ APP_DEBUG=true
 APP_KEY=SomeRandomString
 
 DB_HOST=127.0.0.1
-DB_DATABASE=homestead
-DB_USERNAME=homestead
-DB_PASSWORD=secret
+DB_DATABASE=kheirblogger
+DB_USERNAME=root
+DB_PASSWORD=root
```
Comment (AhmadElsagheer) [inline .env.example:9]: add this file in .gitignore and remove it from the current branch

File: .idea/kheir-blogger.iml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        <CLASSES>
+          <root url="phar://$MODULE_DIR$/composer.phar/" />
+        </CLASSES>
+        <SOURCES>
+          <root url="phar://$MODULE_DIR$/composer.phar/" />
+        </SOURCES>
+      </library>
+    </orderEntry>
+  </component>
+</module>
```
Comment (AhmadElsagheer) [inline .idea/kheir-blogger.iml]: Remove

File: .idea/workspace.xml [hunk 3]
```diff
@@ -0,0 +1,917 @@
...
+    <entry file="file://$PROJECT_DIR$/app/User.php">
+      <provider selected="true" editor-type-id="text-editor">
+        <state vertical-scroll-proportion="0.36349925">
+          <caret line="53" column="5" selection-start-line="53" selection-start-column="5" selection-end-line="53" selection-end-column="5" />
+          <folding />
+        </state>
+      </provider>
+    </entry>
+  </component>
+</project>
```
Comment (AhmadElsagheer) [inline .idea/workspace.xml]: Remove

**Comment by omarradwan** [comment]
in User, the "notifications" function, should it be belongsToMany?
Or hasMany? And what is the difference?

**Comment by AhmadElsagheer** [comment]
belongsToMany. A user can have many notifications and a notification can have man users. It's a many-to-many relation where we use belongsToMany and specifiying the pivot table (in this case, it's user_notifications). hasMany is the inverse of belongsTo in the one-to-many relation

**Comment by AhmadElsagheer** [comment]
belongsToMany. A user can have many notifications and a notification can have man users. It's a many-to-many relation where we use belongsToMany and specifiying the pivot table (in this case, it's user_notifications). hasMany is the inverse of belongsTo in the one-to-many relation

**Comment by AhmadElsagheer** [comment]
belongsToMany. A user can have many notifications and a notification can have man users. It's a many-to-many relation where we use belongsToMany and specifiying the pivot table (in this case, it's user_notifications). hasMany is the inverse of belongsTo in the one-to-many relation

**Comment by AhmadElsagheer** [comment]
belongsToMany. A user can have many notifications and a notification can have man users. It's a many-to-many relation where we use belongsToMany and specifiying the pivot table (in this case, it's user_notifications). hasMany is the inverse of belongsTo in the one-to-many relation

**Comment by AhmadElsagheer** [comment]
belongsToMany. A user can have many notifications and a notification can have man users. It's a many-to-many relation where we use belongsToMany and specifiying the pivot table (in this case, it's user_notifications). hasMany is the inverse of belongsTo in the one-to-many relation

**Comment by AhmadElsagheer** [comment]
belongsToMany. A user can have many notifications and a notification can have man users. It's a many-to-many relation where we use belongsToMany and specifiying the pivot table (in this case, it's user_notifications). hasMany is the inverse of belongsTo in the one-to-many relation

**Pull request merged**